README.md

```diff
@@ -37,18 +44,38 @@ Instructions on how to use them are provided below.
 | ------ | ------ |
 | help | Shows all available commands |
 | hello | Greeting command |
-| add-contact | Adds new contact. Required arguments: name, phone (10 digits) |
+| add-contact | Adds new contact. Required arguments: name, phone (10 digits). Optional argument: email, birthday |
 | change-contact | Changes existing contact. Required arguments: name, old phone, new phone |
+| remove-contact | Delete a contact by name. Required arguments: name |
 | find-contact | Shows contact phones. Required arguments: name or phone search query |
 | all-contacts | Shows all contacts |
 | add-birthday | Adds birthday to contact. Date format: dd.mm.yyyy |
 | show-birthday | Shows contact birthday. Required arguments: name |
-| birthdays | Shows upcoming birthdays |
+| birthdays | Shows upcoming birthdays. Optional argument: number of days |
+| add-email | Adds an email to a contact by name. Required argument: name |
+| show-email | Shows an email of contact by name. Required argument: name |
+| change-email | Changes existing email. Required arguments: name, old email, new email |
 | exit | Exits assistant |
 | close | Alias for Exit command |
 
+## Aditional features
+
+Muad'Dib can store and manage your notes by adding "tags" to your notes + searching notes by these "tags".
+
+| Сommands for notes | Response |
+| ------ | ------ |
+| add-note | Adds new note. Required arguments: title, content. Optional argument: tags |
```

Comment:
So for note we don't use arguments, but use a list of questions for each field. 
for `edit-note` too